website_event_compassion/wizards/event_registration_config_wizard.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models, fields
+
+
+class EventRegistrationConfigWizard(models.TransientModel):
+    _name = 'event.registration.config.wizard'
+    _description = 'wizard that will load a chosen' \
+                   'template for a email to send'
+
+    config = fields.Many2one(
```

Comment:
There is a naming convention on the odoo fields.
For Many2one relation, you should have the suffix `_id`, so the name of the field should be `config_id`.
For Many2many or One2many, the suffix is `_ids`

See https://www.odoo.com/documentation/10.0/reference/guidelines.html#symbols-and-conventions for reference